tools/twix/src/panels/image/overlays/penalty_boxes.rs

```diff
@@ -0,0 +1,41 @@
+use std::str::FromStr;
+
+use color_eyre::Result;
+use communication::client::{Cycler, CyclerOutput};
+use eframe::epaint::{Color32, Stroke};
+use types::Line2;
+
+use crate::{
+    panels::image::overlay::Overlay, twix_painter::TwixPainter, value_buffer::ValueBuffer,
+};
+
+pub struct PenaltyBoxes {
+    penalty_boxes: ValueBuffer,
+}
+
+impl Overlay for PenaltyBoxes {
+    const NAME: &'static str = "Penalty Boxes";
+
+    fn new(nao: std::sync::Arc<crate::nao::Nao>, selected_cycler: Cycler) -> Self {
+        let top_or_bottom = match selected_cycler {
+            Cycler::VisionTop => "top",
+            _ => "bottom",
```

Comment:
```suggestion
            Cycler::VisionBottom => "bottom",
```

Reply:
Non-exhaustive ...
What should I do for other cyclers (that will not actually appear here due to program-logic) ?

Reply:
Changed. Will panic if cycler is not valid.

Added the suggested move from Control to Vision-cyclers to Ideas for next Iteration.
I do not intend to keep working on the camera-calibration subject.